### PR TITLE
added gradle kotlin DSL code samples to the readme file

### PR DESCRIPTION
**Pull request by Jan5366x**
Since Gradle Kotlin DSL is becoming more common we should add related code samples to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +36,38 @@ Alternatively, you can use the `buildscript` DSL:
             classpath 'org.openjfx:javafx-plugin:0.0.7'
         }
     }
-
+
```
Comment (mwkroening) [inline README.md]:
```suggestion

```
Trailing whitespaces

File: README.md [hunk 2]
```diff
@@ -26,13 +36,38 @@ Alternatively, you can use the `buildscript` DSL:
...
-
+    
     apply plugin: 'org.openjfx.javafxplugin'
 
+**Kotlin**
+
+    buildscript {
+        repositories {
+            maven {
+                setUrl( "https://plugins.gradle.org/m2/")
```
Comment (mwkroening) [inline README.md]:
```suggestion
                setUrl("https://plugins.gradle.org/m2/")
```
This whitespace is not gradle style: https://plugins.gradle.org/plugin/org.openjfx.javafxplugin

File: README.md [hunk 3]
```diff
@@ -26,13 +36,38 @@ Alternatively, you can use the `buildscript` DSL:
...
+            }
+        }
+        dependencies {
+            classpath( "org.openjfx:javafx-plugin:0.0.7")
```
Comment (mwkroening) [inline README.md]:
```suggestion
            classpath("org.openjfx:javafx-plugin:0.0.7")
```
Same as above

File: README.md [hunk 4]
```diff
@@ -26,13 +36,38 @@ Alternatively, you can use the `buildscript` DSL:
...
+**Groovy**
+
     javafx {
         modules = [ 'javafx.controls', 'javafx.fxml' ]
     }
+
+**Kotlin**
+
+    javafx {
+        modules.add("javafx.controls")
```
Comment (mwkroening) [inline README.md]: I think the preferred way would be more similar to the Groovy example:
```kotlin
    javafx {
        modules = listOf("javafx.controls", "javafx.fxml")
    }
```

File: README.md (hunk 2 above)
Comment (Jan5366x) [reply README.md]: Ok thank you, I will take care of this!

File: README.md [hunk 5]
```diff
@@ -26,13 +36,35 @@ Alternatively, you can use the `buildscript` DSL:
...
+**Groovy**
+
     javafx {
         modules = [ 'javafx.controls', 'javafx.fxml' ]
     }
+
+**Kotlin**
+
+    javafx {
+        modules = listOf("javafx.controls", "javafx.fxml")
```
Comment (tiainen) [inline README.md]: There's also a method named `modules` that takes a varargs as argument. So you could use this instead:

    modules("javafx.controls", "javafx.fxml")

**Pull request merged**